test/mocks/event/mocks.cc

```diff
@@ -16,6 +16,7 @@ namespace Envoy {
 namespace Event {
 
 MockDispatcher::MockDispatcher() {
+  ON_CALL(*this, initializeStats(_)).WillByDefault(Return());
```

Comment:
This is needed for a handful of existing tests.

Reply:
Are you going to write some new tests? I don't think we need to do anything super fancy with mock time etc., but if we could just run a dispatcher and then check some invariants, e.g. that the histograms are non-empty, the delays are >= expected, I think that would be sufficient.

Reply:
I've written one in `DispatcherImplTest` to verify that we create the expected stats. I haven't written anything to assert that the stats are populated, since I assumed that would be too flaky, but lemme think about that a bit more...